tests/test_oml/test_registry/test_registry.py

```diff
@@ -61,6 +66,7 @@ def get_opt() -> Optimizer:
         ("transforms", TRANSFORMS_REGISTRY, get_transforms, None),
         ("pairwise_model", PAIRWISE_MODELS_REGISTRY, get_pairwise_model, None),
         ("postprocessor", POSTPROCESSORS_REGISTRY, get_postprocessor, None),
+        ("dataset", DATASET_BUILDER_REGISTRY, build_image_dataset, {"dataset_root": MOCK_DATASET_PATH}),
```

Comment:
it should be "datasets_getter" for which you create a folder with example configs
look at the other tests nearby 

I don't understand why this test not failed without this folder of configs. Please, add the folder and update the test so it fails in such cases.

Reply:
Oh, i haven't noticed, that datasets folder in gitignore. Yea, I need to add it to git.

Reply:
oh
yep, and after that and solving conflicts, you can start making tests green

Reply:
Done